_typos.toml

```
@@ -0,0 +1,11 @@
+# Files for typos
+# Instruction:  https://github.com/marketplace/actions/typos-action#getting-started
+
+[default.extend-identifiers]
+NDArray="NDArray" # PyO3
+NdArray="NdArray" # PyO3
```

`NdArray`自体はPyO3関係無く音声合成自体で使っているはずなので、コメントとしてはこちらの方が適切かと思います。

```suggestion
NdArray="NdArray" # onnxruntime::session::NdArray
```

---

あ、ほんとですね！！
上はnumpyだったのでついでにそちらも変えておきます！